content/help/ops/traffic-management/troubleshooting/index.md

```diff
@@ -9,3 +9,36 @@ weight: 5
 * Provide a few general procedures that should be followed to isolate
 
 * Describe high level isolation steps and things to check.
+
+## Pod reachability issues
+
+Usually, the first thing you want to check is whether or not the POD associated with your app has been allocated with a valid IP address. This can be done by examining the POD status with `kubectl get pods -o wide`. In addition, check whether or not the service associated with the app has been properly created with a service IP. Assuming the POD is up and running and IP is allocated, ping the POD IP and make sure it can be pinged. Otherwise, a few things that may be done to isolate the issue:
+
+* Where is it that you’d like to access the app: from inside the same cluster or outside the cluster. If it’s outside the cluster, find out if port-forward is needed, if the right service type is configured for the service associated with the app, etc. If port-forward is needed, use the `kubectl port-forward` command to configure it.
+* can the node where the POD is running be pinged.
+* Check route table to make sure that there is a route to the POD IP: `ip route` or `route -n`.
+* Check to make sure that the return packet path is also properly configured. This involves checking the route tables, iptables, etc, in the node where the POD is running.
+* Check iptables, if necessary, to make sure that packets are not dropped.
+
+## Service naming and Service discovery
+
+A few things that can be checked in this regard:
+
+* Whether or not the service name associated with the app can be properly resolved: login to any other POD’s shell, use `nslookup`, `dig`, etc, to find out if the name can be resolved. Otherwise, check to make sure that
+    * `kube-dns` (if used) is running properly
+    * `/etc/resolve.conf` is properly configured
+* Whether or not the port associated with the app has been opened in the POD: login to the POD or its namespace, and use `netstat` to find out.
```

Comment:
Whether or not the port associaterd wit hthe app has been opened in the POD-> Validate that the port associated with the app has been opened within the pod: ...